notebooks/mping_initial_xai.ipynb

```diff
@@ -0,0 +1,5548 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from PermutationImportance import permutation_importance
</pre><p>Which library is your permutation importance function from? If it's an external library, can you add it to environment.yml?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2es/ptype-physical/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>